scripts/rmOldKibanaIndices.py

```diff
@@ -0,0 +1,29 @@
+import requests
+
+logFile = open('/var/log/probe/KibanaStartup.log', 'w')
+
+res = requests.get('http://localhost:9200/_cat/indices')
+content = res.content
+kibanaIndexNums = []
+
+for indexData in content.split(" "):
+   if ".kibana" not in indexData:
+      continue
+   for term in indexData.split("_"):
+      if ".kibana" in term:
+         continue
+      kibanaIndexNums.append(int(term))
```

Comment:
This could then be one single loop (get indexData) and check if .kibana is in it. If so - append the whole string into an array.